### PR TITLE
cleanup unused kiali chart customizer.

### PR DESCRIPTION
**Pull request by morvencao**
Currently we don't need the kiali chart customizer in the rendering lifecycle, clean this up to reduce the reconcile time.

### CODE REVIEW

**Comment by ostromart** [comment]
@rcernich PTAL if this is ok.

File: pkg/controller/istiocontrolplane/listeners.go [hunk 1]
```diff
@@ -327,49 +325,3 @@ func (c *CitadelChartCustomizer) cleanCitadelResources(obj runtime.Object) error
 	}
 	return nil
 }
-
-// KialiChartCustomizer is a ChartCustomizer for the kiali chart
```
Comment (sdake) [inline pkg/controller/istiocontrolplane/listeners.go]: we probably no longer want to customize KialiChart- but KialiOperator may desire customization...

**Review by sdake — COMMENTED** [review]
not sure this PR is needed.  The Kiali chart is pending deletion - instead this PR may want to focus on the Kiali operator implementation.

**Comment by rcernich** [comment]
The intent when I initially added these hooks was to populate the endpoint details for jaeger and grafana, so they would be configured automatically, without user intervention, e.g. https://github.com/Maistra/istio-operator/blob/maistra-1.1/pkg/controller/servicemesh/controlplane/hooks.go#L62-L181.  The same should apply when using the kiali CR (which is what the linked code modifies).

**Comment by morvencao** [comment]
thanks @rcernich Let's keep it.
/close

**Pull request closed without merging**